_data/identity.yml

```diff
@@ -113,6 +113,16 @@ websites:
       software: Yes
       doc: http://www.onelogin.com/product/strong-authentication/two-factor-authentication/
 
+    - name: PassHub
+      url: https://www.passhub.net/
+      img: PassHub.png
```

Comment:
Everyone else uses lowercase on their images so I suggest you do the same.

Reply:
Actually I did. Initially I used PassHub in the name, but then discovered that everybody else use lowercase, so I renamed the file, deleted the old one, added the new one with all lowercase, and corrected the line in yml